### DISCUSSION TITLE
Dropping support for Catalina (10.15)

### DISCUSSION THREAD

**Discussion by cesarcoatl** [Homebrew/discussions#3129] [Getting started]
With what I've seen in the past, starting with OS X 10.9 (Mavericks), there has been a new OS each year, and if that trend were to continue, macOS 13 might come out this Fall.

Since Homebrew supports only three OS versions at a time, when would you (Homebrew maintainers) expect to drop support for macOS 10.15?

I know that I'll mostly cover all my software needs with MacPorts, but I just want to be prepared.

Thanks!

**Comment by SMillerDev** [comment]
Whenever the next version comes out. We know as much about Apple's plans in this as you do.

**Reply by cesarcoatl** [reply]
Would this be immediately after release?

What have you done in the past?

**Reply by SMillerDev** [reply]
We add a new OS and drop the old one as soon as someone makes the pull request (usually pretty soon after the official release)

**Reply by Bo98** [reply]
A few days before usually. What happens is we stop running CI on that version and stop building bottles for it (replacing that with the new OS), so the "drop date" is really whenever that CI change is made.

The brew CLI is technically supported all the way back to Yosemite, but most things will gradually require building from source and it's up to the community to solve any issues via PRs rather than bug reports.
